vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -2,148 +2,82 @@
             "mcr.microsoft.com/windows/servercore:ltsc2022",
             "mcr.microsoft.com/windows/nanoserver:ltsc2022",
-            "mcr.microsoft.com/oss/kubernetes/pause:3.6-hotfix.20220114",
-            "mcr.microsoft.com/oss/kubernetes/pause:3.9",
-            # CSI. Owner: andyzhangx (Andy Zhang)
-            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.6.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:v2.10.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.5.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:v2.8.0",
-            # azuredisk-csi:v1.27 is only for AKS 1.26+, v1.26 is for other AKS versions
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.2",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.26.3",
-            "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.27.1",
-            # azurefile-csi:v1.26 is only for AKS 1.26+, v1.24 is for other AKS versions
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.24.0",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.24.1",
-            "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.26.2",
-            # Addon of Azure secrets store. Owner: ZeroMagic (Ji'an Liu)
-            "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.3",
-            "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.1",
-            # Azure cloud node manager. Owner: nilo19 (Qi Ni)
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.23.24", # for k8s 1.23.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.11", # for k8s 1.24.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.24.18", # for k8s 1.24.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.5", # for k8s 1.25.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.12", # for k8s 1.25.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.26.0", # for k8s 1.26.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.26.8", # for k8s 1.26.x
-            "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.27.1", # for k8s 1.27.x
-            # OMS-Agent (Azure monitor). Owner: ganga1980 (Ganga Mahesh Siddem)
-            "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.7",
+
             # NPM (Network Policy Manager) & CNS (Container Networking Service) Owner: jaer-tsun (Jaeryn)
             "mcr.microsoft.com/containernetworking/azure-npm:v1.4.34",
-            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.35",
-            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.44")
-} else {
-    throw "No valid windows SKU is specified $windowsSKU"
+            "mcr.microsoft.com/containernetworking/azure-cns:v1.4.35"
```

Comment:
@jaer-tsun Need your help to confirm:
1. could we remove `azure-cns:v1.4.35`? It seems like that `azure-cns:v1.4.44` is used.
2. could we **only** add `azure-npm:v1.4.34` for **WS2022**?